index.html

```diff
@@ -13,226 +13,328 @@
+        <div class="story">
 
-        <p>CSS <img class="rectangle" src="images/Rectangle70.png" alt="rectangle"> Bootstrap
-          <img class="rectangle" src="images/Rectangle70.png" alt="rectangle"> Ruby
-        </p>
-        <a href="#" class="button button-1">See Project</a>
+          <div class="story-head">
+            <h2>Multi-Post Stories</h2>
+          </div>
+
+          <div class="story-paragraph">
+
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+              industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+
+          </div>
+
+          <div class="story-languages">
+
+            <p>CSS <img class="rectangle" src="images/Rectangle70.png" alt="rectangle">
+
+              Bootstrap <img class="rectangle" src="images/Rectangle70.png" alt="rectangle">
+
+              Ruby
+
+            </p>
+
+          </div>
+
+          <div class="story-button">
+
+            <a href="#" class="button button-1">See Project</a>
+
+          </div>
+
+        </div>
 
       </div>
 
+      <div class="contact-us">
 
+        <div class="contact-header">
 
-      <div class="contact">
-        <h1 class="contact-us">Contact us</h1>
+          <h1>Contact Us</h1>
 
-        <p>If you have an application you are interested in developing, a feature that you need built or a project that
-          needs coding. I’d love to help with it
-        </p>
+        </div>
+
+        <div class="contact-paragraph">
+
+          <p>If you have an application you are interested in developing, a feature that you need built or a project
+            that needs coding. I’d love to help with it
+          </p>
+
+        </div>
 
-        <div class="contact-form">
+        <div class="text-field">
 
           <input type="text" name="text-input" placeholder="Text Input"><br>
 
           <input type="email" name="email" placeholder="jeremy@gmail.com"><br><br>
 
-          <textarea class="textarea" placeholder="Enter text here"></textarea>
+        </div>
+
+        <div class="text-area">
+
+          <textarea placeholder="Enter text here"></textarea>
+
+        </div>
+
+        <div class="contact-btn">
 
-          <div class="button2"></div>
           <a href="#" class="button button-1">See Project</a>
+
         </div>
 
-        <div class="btm-rectangle"><img src="images/Rectangle42.png" alt="bottom"></div>
+        <div class="socialbtns btn2">
 
-      </div>
+          <ul>
+            <li><a href="" class="fa fa-lg fa-github"></a></li>
+            <li><a href="" class="fa fa-lg fa-linkedin"></a></li>
+            <li><a href="" class="fa fa-lg fa-angellist"></a></li>
+            <li><a href="" class="fa fa-lg fa-twitter"></a></li>
+            <li><a href="" class="fa fa-lg fa-medium"></a></li>
+          </ul>
+
```

Comment:
- [ ] Please provide the same icons as in fegma template 

![image](https://user-images.githubusercontent.com/74744787/158882012-4019bba4-d28e-4cfb-8b01-cf179b543f9b.png)

Reply:
resolved 👍